go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/davecgh/go-spew v1.1.1
+	github.com/emicklei/go-restful v2.10.0+incompatible // indirect
```

Comment:
This module doesn't need to be written here.
Can you run:
```
go mod vendor
go mod tidy
```

It should remove it.

Reply:
I run both before committing and run them again now, still there :/

Reply:
Maybe you have different go version?
I have go version `go1.16.6`. And on my machine `go mod tidy` removes the line. 

In any case, the line is not needed and you can remove it. Also the change it `vendor/modules.txt` is not needed.

Reply:
done